src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
+/* eslint-disable max-len */
 import React from 'react';
-
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  movie: Movie;
+};
+
+export const MovieCard: React.FC<Props> = ({ movie }) => (
+  <div className="card" data-cy="Movie">
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={movie.imgUrl}
+          alt={movie.title}
+        />
+      </figure>
+    </div>
```

Comment:
```suggestion
export const MovieCard: React.FC<Props> = ({ movie }) => {
   const {
    title,
    description,
    imgUrl,
    imdbUrl,
  } = movie;
  return (
   <div className="card" data-cy="Movie">
     <div className="card-image">
       <figure className="image is-4by3">
         <img
           data-cy="MovieImage"
           src={imgUrl}
           alt={title}
        />
       </figure>
     </div>
```
I suggest to deconstruct the movie so you can see from the beginning which properties you are using.